app/controllers/ExperimentController.php

```diff
@@ -590,6 +609,21 @@ private function getAllowedFileSize()
         }
         return $allowedFileSize;
     }
+
+    private function validateQueueData($computeResourceId, $queue)
+    {
+        $queues = ExperimentUtilities::getQueueDatafromResourceId($computeResourceId);
+        $queueName = $queue['queueName'];
+
+        foreach($queues as $aQueue){
+            if($aQueue->queueName == $queueName) {
+                if($queue['validateQueueData'] <= $aQueue->maxNodes && $queue['maxCPUCount'] <= $aQueue->maxProcessors && $queue['wallTimeLimit'] <= $aQueue->maxRunTime)
```

Comment:
The array keys don't match. Should be `nodeCount` instead of `validateQueueData`, and `cpuCount` instead of `maxCPUCount`.

Reply:
I changed these too. But didn’t commit the new changes.